server/src/db/migrations/20210127112507_dropNameColumnInUsers.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("users", (table) => {
+    table.dropColumn("name")
```

Comment:
Double checking that the User model also reflects this change of not including a `name`